### PR TITLE
[#670] Add test case for soul-plugin-monitor

### PR DESCRIPTION
**Pull request by Wincher**
- [x] You have read the [contribution guidelines](https://dromara.org/en-us/docs/soul/contributor.html).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: soul-plugin/soul-plugin-monitor/src/test/java/org/dromara/soul/plugin/monitor/handler/MonitorPluginDataHandlerTest.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+
+/**
+ * Test case for MonitorPluginHandler.
+ * @author wincher
+ */
+public class MonitorPluginDataHandlerTest {
```
Comment (yu199195) [inline soul-plugin/soul-plugin-monitor/src/test/java/org/dromara/soul/plugin/monitor/handler/MonitorPluginDataHandlerTest.java:34]: add finnal

File: soul-plugin/soul-plugin-monitor/src/test/java/org/dromara/soul/plugin/monitor/handler/MonitorPluginDataHandlerTest.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+import org.dromara.soul.metrics.facade.MetricsTrackerFacade;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+
+/**
+ * Test case for MonitorPluginHandler.
```
Comment (yu199195) [inline soul-plugin/soul-plugin-monitor/src/test/java/org/dromara/soul/plugin/monitor/handler/MonitorPluginDataHandlerTest.java:31]: maybe have a empty line

File: soul-plugin/soul-plugin-monitor/src/test/java/org/dromara/soul/plugin/monitor/handler/MonitorPluginDataHandlerTest.java [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        pluginData = new PluginData("", "", config, 1, true);
+        monitorPluginDataHandler.handlerPlugin(pluginData);
+        assertTrue(MetricsTrackerFacade.getInstance().isStarted());
+    }
+
+    @Test
+    public void testPluginNamed() {
+        assertEquals(PluginEnum.MONITOR.getName(), monitorPluginDataHandler.pluginNamed());
+    }
+
```
Comment (yu199195) [inline soul-plugin/soul-plugin-monitor/src/test/java/org/dromara/soul/plugin/monitor/handler/MonitorPluginDataHandlerTest.java:74]: remove empty line

**Review by yu199195 — APPROVED** [review]

**Comment by Wincher** [comment]
@yu199195, for the failing CI with windows env, I use a Windows PC to run the test code and it works, and it seems like CI with windows env may have an issue when bind port,  **java.net.BindException: Address already in use: bind**, I do it several times with different ports, could you give me a hint on how to deal with this situation.

**Pull request merged**